session/sessiondata.go

```diff
@@ -0,0 +1,101 @@
+package session
+
+import (
+	"os"
+	"strconv"
+	"time"
+
+	goauth2 "golang.org/x/oauth2"
+)
+
+// SessionData is a map respresentation of the session data
+type SessionData map[string]interface{}
```

Comment:
Every time I see 'SessionData' it feels wrong. It **is** the session, and we don't append Data on the end of all our other golang variables (or types) that contain data (ie everything).

This can be just Session, then you have lots of methods (like GetAccessToken) that act on the **Session**, instead of acting on the SessionData (they are one in the same).